credentialz/credentialz.proto

```diff
@@ -592,7 +622,7 @@ enum KeyGen {
   KEY_GEN_SSH_KEY_UNSPECIFIED = 0;
   KEY_GEN_SSH_KEY_TYPE_RSA_2048 = 1;
   KEY_GEN_SSH_KEY_TYPE_ECDSA_P_256 = 2;
-  KEY_GEN_SSH_KEY_TYPE_ECDSA_P_521 = 3;
+  KEY_GEN_SSH_KEY_TYPE_ECDSA_P_512 = 3;
```

Comment:
Explicitly noting that this is still incorrect since the prior conversation is marked as resolved (should be 521, not 512) :)

Reply:
had overwritten push